srcpkgs/elisa/template

```diff
@@ -1,19 +1,23 @@
 # Template file for 'elisa'
 pkgname=elisa
-version=24.02.2
-revision=2
+version=24.05.1
+revision=1
 build_style=cmake
+build_helper=qemu
```

Comment:
You are doing it wrong again. It only need `qt6-declarative-host-tools` in `hostmakedepends`.